test/unit/org/apache/cassandra/ServerTestUtils.java

```diff
@@ -189,6 +190,16 @@ public static void cleanupSavedCaches()
         cleanupDirectory(DatabaseDescriptor.getSavedCachesLocation());
     }
 
+    public static EmbeddedCassandraService startEmbeddedCassandraService() throws IOException
```

Comment:
I am wondering whether this is not a bit of a confusing name. We already have a start method in EmbeddedCassandraService.

Reply:
Why confusing? `start` is instance method in `EmbeddedCassandraService`. This is a different class, dedicated solely for setting up Cassandra instance for testing purposes